home/home_2.1/TestCases.txt

```diff
@@ -0,0 +1,44 @@
+Test Case 1:
+Step 1: User starts a program; Expected Result: Program starts
+Step 2: User should enter a name "Alice"; Expected Result: Program should display "Hello, Alice"
+Step 3: Program creates a file "user_stats.txt"; Expected Result: Program should create a file "user_stats.txt"
+Step 4: Observe the output of the file "user_stats.txt"; Expected Result: File contains a line "Alice 1"
```

Comment:
Consider more straightforward formulation for step; something like: Run command cat user_stats.txt ER: Output contains a single line Aliice 1

Why it is important? It may save a lot of time just by copying existing command and paste it during test run instead of manual typing or opening via editor.